src/grisp_tools_io.erl

```diff
@@ -108,9 +125,16 @@ trim(Str) -> string:strip(rebar_utils:to_list(Str)).
 trim(Str, Dir, [Chars|_]) -> string:strip(rebar_utils:to_list(Str), Dir, Chars).
 -endif.
 
-say(State, Say, Args) when is_list(Args) ->
-    Event = {say, io_lib:format(lists:flatten([Say, "~n"]), Args)},
-    grisp_tools_util:event(State, Event);
-say(State, Say, Args) ->
-    Event = {say, io_lib:format(lists:flatten([Say, "~n"]), [Args])},
+say(State, Say) ->
+    Event = {say, Say},
     grisp_tools_util:event(State, Event).
+
+
+% @doc Add foreground coloring to the given text using ANSI ecape code
+% See: https://en.wikipedia.org/wiki/ANSI_escape_code#8-bit
+% @end
+-spec color(Code, Text) -> string() when
+      Code :: 0..255,
+      Text :: string().
+color(Code, Text) ->
+    io_lib:format("\033[38;5;~pm~s\033[m", [Code, Text]).
```

Comment:
Hope this doesn't mess up windows terminals

Reply:
I just tested on my Windows computer and it worked with the basic terminal.
Apparently, ANSI sequences are now activated by default on Windows 10/11 (it wasn't the case before)